src/github.com/getlantern/flashlight/main/pac.go

```diff
@@ -76,31 +76,27 @@ func genPACFile() {
 	}
 	formatter :=
 		`var bypassDomains = %s;
-		function FindProxyForURL(url, host) {
-			if (isPlainHostName(host) // including localhost
-			|| shExpMatch(host, "*.local")) {
+	function FindProxyForURL(url, host) {
```

Comment:
Should we hardcode `ui.lantern.io` here to make sure it always passes through Lantern or that's not required?